GAE/src/resources/caddisfly/caddisfly-tests.json

```diff
@@ -0,0 +1,154 @@
+{
```

Comment:
I don't think it is necessary - the app is designed to fail gracefully - it will just report 'unknown test'. If we had a version number it would basically do the same thing.